routes/api/auth.js

```diff
@@ -177,6 +177,18 @@ module.exports = {
          * @apiSuccessExample {json} Success-Response:
          *      {"message": "Successfully resent confirmation email", "data": {}}
          * 
+         * @apiError {string} message Error message
+         * @apiError {object} data empty
+         * @apiSuccessExample {json} Error-Response:
+         *       HTTP/1.1 422
+         *      {"message": "Account already confirmed, "data": {}}
+         * 
+         * @apiError {string} message Error message
+         * @apiError {object} data empty
+         * @apiSuccessExample {json} Error-Response:
```

Comment:
change this to apiErrorExample (see <http://apidocjs.com/#param-api-error-example>)